test/test_http_instrumentation.py

```diff
@@ -685,15 +693,16 @@ def test_record_post_data_ajax_no_key_value(self):
         """Test AJAX payloads that are not in the key=value form."""
         post_format = "noKeyValue"
         db = self.visit("/post_request_ajax.html?format=" + post_format)
-        post_body = self.get_post_request_body_from_db(db)
-        assert post_body == "test@example.com + name surname"
+        post_body = self.get_post_request_body_from_db(db, True)
+        assert post_body == "test@example.com + name surname".encode('utf8')
```

Comment:
nit: You should use `post_body.decode('utf8')` here as you do above, rather than encode the literal.

Reply:
Done